### PR TITLE
Api stuff

### PR DESCRIPTION
**Pull request by Rikshy**
Hi there!

i made some extensions for the api :)

i need it for my small addon mod
https://github.com/Rikshy/AntiqueAtlasExtras

### CODE REVIEW

File: src/main/java/hunternif/mc/atlas/api/MarkerAPI.java [hunk 1]
```diff
@@ -47,8 +49,10 @@ void putMarker(World world, boolean visibleAhead, int atlasID,
 	 * @param label			text label to be displayed on mouseover.
 	 * @param x				block coordinate
 	 * @param z				block coordinate
+	 *
+	 * @return returns the marker id. -1 if failed or client
 	 */
-	void putGlobalMarker(World world, boolean visibleAhead,
+	int putGlobalMarker(World world, boolean visibleAhead,
```
Comment (liach) [inline src/main/java/hunternif/mc/atlas/api/MarkerAPI.java]: You should just return the marker.

**Comment by Rikshy** [comment]
Actually I thought about that, forgot why I didn't tho. But you are right, Yea, I could just do that.

**Comment by liach** [comment]
Everything else looks great.

**Comment by Rikshy** [comment]
there ya go :P

**Review by liach — APPROVED** [review]

**Comment by Kenkron** [comment]
I'll be on it as soon as I can. Promise!

**Comment by Rikshy** [comment]
Is there a more specific ETA? :P

**Comment by Kenkron** [comment]
I'll do it TODAY! No more excuses! Thanks for the reminder. Besides, I'm excited to see the extra stuff.

**Comment by Rikshy** [comment]
You still need to upload a new version

**Comment by Rikshy** [comment]
Bump

**Pull request merged**